tools/depends/target/config.site.in

```diff
@@ -106,6 +95,10 @@ fi
 # Python3 build for appletvos
 if test "@target_platform@" = "appletvos"; then
   ac_cv_func_sigaltstack=no
+  if test "${PACKAGE_NAME}" = "gnutls"; then
+    ac_cv_func_vfork_works=no
+    ac_cv_func_fork=no
```

Comment:
Does `fork` and `vfork` work in general on appletvos?

Reply:
No, it appears not.

Reply:
In that case, why is it only disabled for a package named gnutls?